model/otel/metrics.yaml

```diff
@@ -206,6 +206,63 @@ groups:
       - ref: error.type
         examples: ["rejected", "timeout", "500", "java.net.UnknownHostException"]
 
+  - id: metric.otel.sdk.exporter.metric.inflight
+    type: metric
+    metric_name: otel.sdk.exporter.metric.inflight
+    stability: development
+    brief: "The number of metric data points which were passed to the exporter, but that have not been exported yet (neither successful, nor failed)"
+    note: |
+      For successful exports, `error.type` MUST NOT be set. For failed exports, `error.type` must contain the failure cause.
+    instrument: updowncounter
+    unit: "{data_point}"
+    attributes:
+      - ref: otel.component.type
+      - ref: otel.component.name
+      - ref: server.address
+        requirement_level:
+          recommended: when applicable
+      - ref: server.port
+        requirement_level:
+          recommended: when applicable
+
+  - id: metric.otel.sdk.exporter.metric.exported
+    type: metric
+    metric_name: otel.sdk.exporter.metric.exported
+    stability: development
+    brief: "The number of metric data points for which the export has finished, either successful or failed"
+    note: |
+      For successful exports, `error.type` MUST NOT be set. For failed exports, `error.type` must contain the failure cause.
+      For exporters with partial success semantics (e.g. OTLP with `rejected_data_points`), rejected data points must count as failed and only non-rejected data points count as success.
+      If no rejection reason is available, `rejected` SHOULD be used as value for `error.type`.
+    instrument: counter
+    unit: "{data_point}"
+    attributes:
+      - ref: otel.component.type
+      - ref: otel.component.name
+      - ref: server.address
+        requirement_level:
+          recommended: when applicable
+      - ref: server.port
+        requirement_level:
+          recommended: when applicable
+      - ref: error.type
+        examples: ["rejected", "timeout", "500", "java.net.UnknownHostException"]
+
+  - id: metric.otel.sdk.metricreader.collection.duration
```

Comment:
nit: based on the naming guidance, we should probably call it `metric_reader` since these are two inseparable words.

/cc @trask

Reply:
Fixed in https://github.com/open-telemetry/semantic-conventions/pull/2016/commits/39378f6fd0bd8f6abe9fc7b5e6122d26d4722e56